### ISSUE TITLE
Awesome

### ISSUE THREAD

**Issue by rgbkrk** [yuvipanda/jupyterhub-nginx-chp#1]
Any ability to share what it's been like to run this in lieu of CHP? I've been curious about this approach for a while and still don't have the background with both Lua and Lua+nginx.

**Comment by yuvipanda** [comment]
:D It's all new, but so far it's provided me all the same features of CHP, without having to setup things from npm. There might be performance improvements too, but hard to tell without https://github.com/jupyter/configurable-http-proxy/issues/49. 

It's still missing a bunch of features (SSL support, proper logging, documentation!) so I'll probably announce it on the list once I add those (hopefully over the weekend)

**Comment by ryanlovett** [comment]
@rgbkrk We've been using this on a fairly busy hub and it has eliminated 503 proxy errors. (knock on wood) So far so good.

**Comment by yuvipanda** [comment]
yeah, that was the deal for us too. random 503s were completely eliminated when we switched to this.
